src/components/VFilters/VFilterChecklist.vue

```diff
@@ -20,40 +20,45 @@
         <VLicense v-if="filterType === 'licenses'" :license="item.code" />
         <template v-else>{{ itemLabel(item) }}</template>
       </VCheckbox>
-      <button
-        v-if="filterType === 'licenses'"
-        :ref="`${item.code}licenseIcon`"
-        :aria-label="$t('browse-page.aria.license-explanation')"
-        class="text-dark-charcoal-70 appearance-none"
-        type="button"
-        @click.stop="toggleLicenseExplanationVisibility(item.code)"
-      >
-        <VIcon :icon-path="helpIcon" />
-      </button>
+
+      <!-- License explanation -->
+      <VPopover v-if="filterType === 'licenses'">
+        <template #trigger="{ a11yProps }">
+          <button
+            v-bind="a11yProps"
+            :ref="`${item.code}licenseIcon`"
+            :aria-label="$t('browse-page.aria.license-explanation')"
+            class="text-dark-charcoal-70 appearance-none"
```

Comment:
Is `appearance-none` redundant? I think tailwind already applies that to `button` elements in its reset stylesheet.

Reply:
Tailwind only applies these:
```css
-webkit-appearance: button;
background-color: transparent;
```
But your assessment that `appearance-none` doesn't do anything is correct. Removing or keeping it seems to have no effect on the UI.

Reply:
Resolved in f69df84.

Reply:
Ah! I was thinking of `webkit-appearance`, by bad!

Did you test that in different browser by any chance? I wonder if `appearance-none` was doing any resets for Safari for example?